app/styles/ui/drag-elements/_cherry-pick-commit.scss

```diff
@@ -55,6 +55,7 @@
       display: flex;
       align-items: center;
       justify-content: center;
+      white-space: nowrap;
```

Comment:
Why did you need this?

Reply:
![image](https://user-images.githubusercontent.com/75402236/115246952-585ebc00-a0f4-11eb-81ca-26587bf8f27a.png)
Other wise got weird wrapping.. sometimes even the Copy To would split to two lines... and this was current problem with windows if you had a really long branch name in comparison to the commit message.